src/2_RTB/PianoDiQualifica/content.typ

```diff
@@ -0,0 +1,153 @@
+
+
+== Scopo 
+
+Il presente documento ha i seguenti scopi:
+
+          - Definizione standard di qualità;
+          - Definizione metriche e criteri quantitativi per valutare l'efficacia del processo e del prodotto;
+          - Gestione dei test: definizione di test, la pianificazione delle attività di test, la valutazione dei risultati.
+          - Verifica e validazione del prodotto includendo test di unità, di integrazione, di sistema, di accettazione;           
+          - Gestione dei Difetti e delle Anomalie riscontrate durante le attività di verifica;
+          - Miglioramento continuo attraverso l'adattamento del piano in base alle esperienze acquisite e alle modifiche nei requisiti del progetto. 
+
+#pagebreak()
+= Qualità di Processo 
+
+== Obiettivi 
+
+Si riferisce all'efficacia ed efficienza dei processi e delle metodologie impiegate nello sviluppo, testing e mantenimento del software. Essa abbraccia l'insieme di attività, pratiche e metodi utilizzati lungo l'intero ciclo di vita dello sviluppo del software al fine di garantire che il prodotto finale soddisfi i requisiti specificati e venga consegnato puntualmente e entro il budget previsto. In questo contesto, la qualità di processo è fondamentale per assicurare un'implementazione coerente, misurabile e ottimizzata delle fasi di sviluppo, contribuendo a conseguire gli obiettivi di qualità complessivi.
+
+
+== Processi Primari  
+
+=== Fornitura 
+
+Le metriche di progetto usate nella fornitura sono misure utilizzate per valutare diversi aspetti del processo di sviluppo e del prodotto risultante. Alcune delle metriche comuni utilizzate nella fornitura di progetti software includono:
+
+  - *TCP*: _Total Cost Project_  indica la somma totale dei costi associati al progetto;
+  - *BAC*: _Budget At Completion_  indica il budget totale pianificato per il completamento del progetto;
+  - *EV*: _Earned Value_  rappresenta il valore prodotto dal progetto ossia il valore dei _deliverable_ rilasciati fino al momento della misurazione in seguito alle attività    svolte;
+      - Formula:  $"EV" = "Percentuale di completamento del lavoro" dot "BAC"$.
+  - *PV*: _Planned Value_ Il valore del lavoro pianificato fino a un dato momento;
+        - Formula: $"PV" = "Percentuale di pianificazione del lavoro" dot "BAC"$.
+  - *AC*: _Actual Cost_ Il costo effettivo sostenuto fino a un dato momento;
+  - *CPI*:_Cost Performance Index_ Misura l'efficienza del costo del lavoro svolto fino a un dato momento;
+          - Formula: $"CPI" = "EV" / "AC"$.
+  - *SPI*: _Schedule Performance Index_ Misura l'efficienza del tempo rispetto alla pianificazione;
```

Comment:
manterrei la formattazione precedente in cui la prima lettera della parola dopo il corsivo rimane minuscola per questo e i punti successivi